packages/camera/camera_windows/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.2.0+1
+
+**BREAKING CHANGES**:
+  * `CameraException.code` now has value `"CameraAccessDenied"` if camera access permission was denied.
+  * `CameraException.code` now has value `"camera_error"` if error occurs during capture.
```

Comment:
This may not be clear, but a camera permission error during capture will still have a code of `CameraAccessDenied`. Let me know if you have suggestions on how to word this better.